specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -1476,13 +1534,30 @@
           "description": "The timestamp when the chat thread was deleted. The timestamp is in ISO8601 format: `yyyy-MM-ddTHH:mm:ssZ`.",
           "type": "string",
           "example": "2020-10-30T10:50:50Z"
-        },
-        "participants": {
-          "description": "Chat participants.",
+        }
+      }
+    },
+    "CreateChatThreadErrors": {
+      "type": "object",
+      "properties": {
+        "invalidParticipants": {
+          "description": "The participants that failed to be added to the chat thread.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/ChatParticipant"
-          }
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "CreateChatThreadResult": {
+      "type": "object",
+      "properties": {
+        "chatThread": {
```

Comment:
"description": "The thread created."
If we don't add descriptions, the sdks side complains with warnings.

Same for errors

Reply:
This is odd, the type already has xml documentation there.

![image](https://user-images.githubusercontent.com/665804/100776991-11c52b00-33ba-11eb-8ad7-7b2769380960.png)

Reply:
This is fixed now, the description is under the "ChatThread" definition.